eslint-plugin/README.md

```diff
@@ -60,6 +60,7 @@ Add `@ecocode` to the `plugins` section of your `.eslintrc`, followed by rules c
 | :------------------------------------------------------------------------------------- | :--------------------------------------------------------- | :- |
 | [avoid-high-accuracy-geolocation](docs/rules/avoid-high-accuracy-geolocation.md)       | Avoid using high accuracy geolocation in web applications. | ✅  |
 | [no-import-all-from-library](docs/rules/no-import-all-from-library.md)                 | Should not import all from library                         | ✅  |
+| [no-missing-print-css](docs/rules/no-missing-print-css.md)                             | Warn when no print CSS is provided                         | ✅  |
```

Comment:
I find the double negative a bit hard to read (no missing), maybe just name the rule "provide-css-print"?
```suggestion
| [provide-css-print](docs/rules/provide-css-print.md)                                   | Enforce providing a print stylesheet                     | ✅  |
```